docs/operate/monitoring/tracing.mdx

```diff
@@ -20,19 +20,20 @@ Set up the [OTLP exporter](https://github.com/open-telemetry/opentelemetry-colle
 
 ### Exporting traces to Jaeger
```

Comment:
Maybe add to the above section ("Setting up OTLP exporter") a remark that Restate supports the grpc and http (including the TLS variants) protocol and how to choose/configure it.